sale_order_report_with_note/views/report_saleorder_qweb.xml

```diff
@@ -0,0 +1,167 @@
+                            </xpath>
+                        </template>
+                        <div class="row">
+                            <div class="col-xs-6">
+                                <div class="Address">
+                                    <strong>Shipping address :</strong>
+                                    <div t-field="o.partner_shipping_id" 
+                                        t-field-options='{"widget": "contact", "fields": ["address", "name", "phone", "fax"], "no_marker": true}'/>
+                                </div>
+                                <div class="Address">
+                                    <strong >Invoice address:</strong>
+                                        <div t-field="o.partner_invoice_id"
+                                            t-field-options='{"widget": "contact", "fields": ["address", "name", "phone", "fax"], "no_marker": true}'/>
+                                </div>
+                            </div>
+                            <div class="col-xs-5 col-xs-offset-1">
+                                <div t-field="o.partner_id"
+                                    t-field-options='{"widget": "contact", "fields": ["address", "name", "phone"], "no_marker": true}' />
+                            </div>
+                        </div>
+                        <h3 style="clear:both; padding-top: 20px;">
+                            <span t-if="o.state not in ['draft','sent']">Order N° </span>
+                            <span t-if="o.state in ['draft','sent']">Quotation N° </span>
+                            <span t-field="o.name"/>
+                        </h3>
+                        <table class="table table-bordered table-condensed" width="100%">
+                            <tr>
+                                <th style="text-align:center" class="date" t-if="o.state not in ['draft','sent']">Quotation Date</th>
+                                <th style="text-align:center" class="date" t-if="o.state in ['draft','sent']">Date Ordered</th>
+                                <th style="text-align:center" >Your Reference</th>
+                                <th style="text-align:center">Salesman</th>
+                                <th style="text-align:center">Payment Term</th>
+                            </tr>
+                            <tr>
+                                <td class="date"><t t-esc="o.date_order"/></td>
+                                <td><t t-esc="o.client_order_ref"/></td>
+                                <td style="text-align:center"><t t-esc="o.user_id.name"/></td>
+                                <td style="text-align:center"><t t-esc="o.payment_term_id.name"/></td>
+                            </tr>
+                        </table>
+                        <t t-if="o.note1"><t t-raw="o.note1"/></t>
+                            <table class="list_main_table" width="100%">
```

Comment:
A layouted sale order would not be possible anymore. It is still working, but the configuration in Report Layout Categories are ignored.
It would be necessary to add 
<t t-foreach="doc.order_lines_layouted()" t-as="page"> 
and the corresponding notes into the view.

Reply:
@FrankKub I am not really sure the use of the layout you are mentioning.

What is the purpose for it?